<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Interface</class>
 <widget class="QMainWindow" name="Interface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>877</width>
    <height>617</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IA - Buscas</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QComboBox" name="cbOrigem">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelInicial">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Selecione o nó inicial:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelFinal">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>10</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Selecione o nó final</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>480</y>
      <width>831</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>450</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Mostrar Caminho</string>
    </property>
   </widget>
   <widget class="QComboBox" name="metodos">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>DFS</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>BFS</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="labelInicial_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Selecione o Método</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbFinal">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>30</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="start">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>80</y>
      <width>481</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Iniciar</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>877</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGrafos">
    <property name="title">
     <string>Grafos</string>
    </property>
    <addaction name="actionLoad"/>
   </widget>
   <addaction name="menuGrafos"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLoad"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionLoad">
   <property name="text">
    <string>Carregar</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
